prisma/schema.prisma

```diff
@@ -9,14 +9,77 @@ datasource db {
 
 model User {
   id       String     @id @default(auto()) @map("_id") @db.ObjectId
-  email    String     @unique
-  name     String?
-  activity Activity[]
+  name     String
+  email    String     
+  password String
+  wishList String[]
+  activity Activity[] @relation("userActivity")
+  venueId Venue? @relation("userVenue")
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
 }
 
 model Activity {
   id     String @id @default(auto()) @map("_id") @db.ObjectId
   name   String
-  user   User   @relation(fields: [userId], references: [id])
+  description String
+  image String
+  venueId Venue? @relation("activityVenue")
+  duration Int
+  category Category @default(Indoor)
+  numPlayers Int
+  minPlayers Int
+  maxPlayers Int
+  difficultRating DifficultyRating @default(Normal)
+  likeReview Float
+  user   User?   @relation("userActivity", fields: [userId], references: [id])
   userId String @db.ObjectId
```

Comment:
userId not exist in Activities Document

Reply:
This is the FK connection between User and Activity.